src/components/ui/label.tsx

```
@@ -0,0 +1,26 @@
+"use client"
+
+import * as React from "react"
```

ESLint

[error] Replace `"react"` with `'react'`

(prettier/prettier)